Contact/ContactRepository.cs

```diff
@@ -5,13 +5,17 @@
 
 namespace Nate.ContactApp
 {
-    public class ContactRepository
+    public class ContactRepository : IContactRepository
     {
         public List<Contact> ContactList;
```

Comment:
this could be static if you want to treat this as a faux database until you wire up a real one.

Reply:
Should probably be private as you don't want external classes accessing stuff you are only using externally.